docs/source/api_reference/transformations.rst

```diff
@@ -333,6 +333,14 @@ Filtering and denoising
 
     BKFilter
 
+.. currentmodule:: sktime.transformations.series.hpfilter
```

Comment:
actually, could you kindly move the `Filter` to the top?
Things should roughly be ordered by how common they are, I think `Filter` (generic band pass) is the most common filter people might be looking for. The others are more finance/econ specific afaik.